controllers/licenseissuer/internal/controller/notification_controller.go

```diff
@@ -146,16 +160,15 @@ func (r *NotificationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 // SetupWithManager sets up the controller with the Manager.
 func (r *NotificationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	r.logger = ctrl.Log.WithName("NotificationReconcile")
-	r.NotificationMgr = cloud.NewNotificationManager()
-	r.Users = cloud.UserCategory{}
+	r.NotificationMgr = issuer.NewNotificationManager()
+	r.Users = issuer.UserCategory{}
 	Predicate := predicate.NewPredicateFuncs(func(object client.Object) bool {
-		return object.GetName() == string(cloud.ClientStartName) &&
-			object.GetNamespace() == string(cloud.Namespace) &&
+		return object.GetName() == string(issuer.ClientStartName) &&
+			object.GetNamespace() == string(issuer.Namespace) &&
 			object.GetLabels() != nil &&
-			object.GetLabels()[string(cloud.IsRead)] == cloud.FALSE &&
-			object.GetLabels()[string(cloud.ExternalNetworkAccessLabel)] == string(cloud.Enabled)
+			object.GetLabels()[string(issuer.NotificationLable)] == string(issuer.FALSE)
```

Comment:
use a func and make this ez to read.

Reply:
For code that does not have reusability and has simple logic, using a closure function is a good choice. There is no need for additional function calls.

Reply:
then make it ez to read.